Hardware/HardwareTests/UnderlyingError+StripeTests.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+@testable import Hardware
+import StripeTerminal
+
+final class UnderlyingError_StripeTests: XCTestCase {
+
+    func test_stripe_offlineDeclineCode_used_to_determine_decline_reason() {
```

Comment:
Just curious: Why the _offline_ mention in the test name?

Reply:
Because I didn't fully correct it from a previous reference to `offlineDeclineReason`, which is a different Stripe error key. I've fixed it in e81f64f, thanks for pointing that out!